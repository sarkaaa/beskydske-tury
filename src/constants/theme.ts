import * as colors from "./colors";
import * as sizes from "./sizes";

const theme = {
  colors,
  sizes
};

export default theme;