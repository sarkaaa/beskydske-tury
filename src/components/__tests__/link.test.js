import React from "react"
import renderer from "react-test-renderer"
import { ThemeProvider } from "styled-components"
import "jest-styled-components"

import StyledLink from "../link"
import theme from "../../constants/theme"

const Wrapper = ({ children }) => (
  <ThemeProvider theme={theme}>{children}</ThemeProvider>
)

test("Link renders correctly", () => {
  const tree = renderer
    .create(
      <Wrapper>
        <StyledLink href="/">Domovská stránka</StyledLink>
      </Wrapper>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
