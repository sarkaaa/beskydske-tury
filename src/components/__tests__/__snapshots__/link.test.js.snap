// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
.c0 {
  position: relative;
  display: inline-block;
  margin: 0 0.1rem;
  overflow: hidden;
  vertical-align: bottom;
  z-index: 0;
  -webkit-transition: ease-out .4s;
  transition: ease-out .4s;
}

.c0:hover a {
  color: #0D7419;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c0:hover::before {
  -webkit-transform: translateY(0);
  -ms-transform: translateY(0);
  transform: translateY(0);
  -webkit-transition: -webkit-transform .25s ease-out;
  -webkit-transition: transform .25s ease-out;
  transition: transform .25s ease-out;
  border-radius: 0.25rem;
}

.c0 a {
  color: #4A4A4A;
  font-family: 'Lato',sans-serif;
  font-weight: 400;
  line-height: 1;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
}

<div
  className="c0"
>
  <a
    href="/"
  >
    Domovská stránka
  </a>
</div>
`;
